server/schemas/mutations/add-expense.mutation.js

```diff
@@ -25,11 +25,11 @@ export default {
   },
   resolve: async (_, { input }) => {
```

Comment:
By the way, been thinking of asking about the `_`, but I always forget. What does it do if I may ask?

Reply:
It's just a way of ignoring the first argument to the function @oreoluwade